src/native/fs/fs_napi.cpp

```diff
@@ -99,10 +100,22 @@ struct FSWorker : public Napi::AsyncWorker
     virtual void Work() = 0;
     void Execute() override
     {
+        using std::chrono::duration_cast; 
+        using std::chrono::milliseconds;
+        using std::chrono::system_clock;
+
         DBG1("FS::FSWorker::Execute: " << _desc << DVAL(_uid) << DVAL(_gid) << DVAL(_backend));
+        if (_desc.find("FileWritev") != std::string::npos ) {
+            thresholdInMilliSec = 100;
+        }
```

Comment:
This is not the way to change behavior of inherited classes.
Instead, use a method called `set_work_threashold(...)` and call it in the ctor of FileWritev.